packages/stark-core/src/modules/routing/services/routing.service.spec.ts

```diff
@@ -2,10 +2,8 @@ import { Component, NgModuleFactoryLoader, NO_ERRORS_SCHEMA, SystemJsNgModuleLoa
 import { TestBed, fakeAsync, tick, inject } from "@angular/core/testing";
 import { Ng2StateDeclaration, UIRouterModule } from "@uirouter/angular";
 import { StateDeclaration, StateObject, StateService, TransitionService, UIRouter } from "@uirouter/core";
-import { switchMap } from "rxjs/operators/switchMap";
-import { catchError } from "rxjs/operators/catchError";
-import { tap } from "rxjs/operators/tap";
-import { _throw as observableThrow } from "rxjs/observable/throw";
+import { catchError, switchMap, tap } from "rxjs/operators";
```

Comment:
Maybe you can add a FIXME for the switchMap operator to refactor the code where is used so that we fully aling with the RXJS Migration guide.

Reply:
done